tests/test_redis_session.py

```
@@ -0,0 +1,251 @@
+# tests/test_redis_session.py
+import pytest
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)